sops/__init__.py

```diff
@@ -820,6 +840,10 @@ def encrypt(value, key, aad=b'', stash=None, digest=None):
     if digest:
         digest.update(value)
 
+    # if we don't want to encrypt, then return the value
+    if unencrypted:
+        return value
+
```

Comment:
If we know in advance we don't want to encrypt the value, it would be preferable to just not enter `encrypt()`